mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/jobcluster/JobClusterActor.java

```diff
@@ -1895,6 +1906,38 @@ private long getExpireAcceptedDelayMs() {
         return empty();
     }
 
+
+    private Optional<JobDefinition> createJobDefinitionInheritingInstanceCount(
+            final JobDefinition givenJobDefn,
+            final List<JobInfo> existingJobsList) {
+        if(logger.isTraceEnabled()) { logger.trace("Enter createJobDefinitionInheritingInstanceCount"); }
```

Comment:
do we need this or was this meant for debugging?

Reply:
I see this is the widely used pattern in mantis codebase to provide more detailed debugging traces. I think by default it's turned off.